### PR TITLE
docs: Mark OEP-48 as Approved and OEP-23 as replaced

### PR DESCRIPTION
**Pull request by xitij2000**
OEP-48 deals with customising branding for MFEs usin an npm package override.
This mechanism for branding is now supported across all MFEs, and is also
supported by both the configuration repo, and Tutor. As such this PR updates its
status from Provisional to Accepted.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @xitij2000!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: oeps/architectural-decisions/oep-0048-brand-customization.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ OEP-48: Brand Customization
    * - Arbiter
      - Kshitij Sobti <kshitij@opencraft.com>
    * - Status
-     - Provisional
+     - Approved
```
Comment (kdmccormick) [inline oeps/architectural-decisions/oep-0048-brand-customization.rst:18]:
```suggestion
     - Accepted
```

**Review by kdmccormick — COMMENTED** [review]
Thanks for making this change @xitij2000 , I agree it's time to mark this as complete. Could you:

* mark OEP-23 as "Replaced"
* add a changelog entry to the bottom of this OEP as well as OEP-23
* post this PR in the #open-edx-proposals channel and wait a week, just to give folks a chance to raise their hand if they disagree

**Comment by xitij2000** [comment]
> Thanks for making this change @xitij2000 , I agree it's time to mark this as complete. Could you:
> 
>     * mark OEP-23 as "Replaced"
> 
>     * add a changelog entry to the bottom of this OEP as well as OEP-23
> 
>     * post this PR in the #open-edx-proposals channel and wait a week, just to give folks a chance to raise their hand if they disagree

Sorry for the delay. I've updated the PR.

**Review by arbrandes — APPROVED** [review]

**Review by kdmccormick — APPROVED** [review]
Thanks again!

**Comment by itsjeyd** [comment]
Hey @kdmccormick and @arbrandes! Now that this PR is ready to go, would it be possible to move forward with merging it? I'm not sure who has permission to hit the button; please let me know if there's someone else I should ping.

CC @mphilbrick211

**Comment by openedx-webhooks** [comment]
@xitij2000 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/open-edx-proposals&entry.867055334=https://github.com/openedx/open-edx-proposals/pull/395&entry.1484655318=https://github.com/xitij2000&entry.752974735=2022-10-07+07:21&entry.1917517419=2022-11-14+14:47&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**